compose/gettingstarted.md

```diff
@@ -98,73 +96,142 @@ This Compose file defines two services, `web` and `redis`. The web service:
-need to share the drive or location of the Dockerfile and volume you are using.
-If you get runtime errors indicating an application file is not found, a volume
-mount is denied, or a service cannot start, try enabling file or drive sharing.
-Volume mounting requires shared drives for projects that live outside of
-`C:\Users` (Windows) or `/Users` (Mac), and is required for _any_ project on
-Docker for Windows that uses [Linux
-containers](/docker-for-windows/#switch-between-windows-and-linux-containers-beta-feature).
-For more information, see [Shared
-Drives](../docker-for-windows/#shared-drives) on Docker for Windows,
-[File sharing](../docker-for-mac/#file-sharing) on Docker for Mac, and
-the general examples on how to [Manage data in
-containers](../engine/tutorials/dockervolumes.md).
-
 ## Step 4: Build and run your app with Compose
 
-1. From your project directory, start up your application.
+1.  From your project directory, start up your application.
+
+    ```
+    $ docker-compose up
+    Creating network "composetest_default" with the default driver
+    Creating composetest_web_1 ...
+    Creating composetest_redis_1 ...
+    Creating composetest_web_1
+    Creating composetest_redis_1 ... done
+    Attaching to composetest_web_1, composetest_redis_1
+    web_1    |  * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
+    redis_1  | 1:C 17 Aug 22:11:10.480 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
+    redis_1  | 1:C 17 Aug 22:11:10.480 # Redis version=4.0.1, bits=64, commit=00000000, modified=0, pid=1, just started
+    redis_1  | 1:C 17 Aug 22:11:10.480 # Warning: no config file specified, using the default config. In order to specify a config file use redis-server /path/to/redis.conf
+    web_1    |  * Restarting with stat
+    redis_1  | 1:M 17 Aug 22:11:10.483 * Running mode=standalone, port=6379.
+    redis_1  | 1:M 17 Aug 22:11:10.483 # WARNING: The TCP backlog setting of 511 cannot be enforced because /proc/sys/net/core/somaxconn is set to the lower value of 128.
+    web_1    |  * Debugger is active!
+    redis_1  | 1:M 17 Aug 22:11:10.483 # Server initialized
+    redis_1  | 1:M 17 Aug 22:11:10.483 # WARNING you have Transparent Huge Pages (THP) support enabled in your kernel. This will create latency and memory usage issues with Redis. To fix this issue run the command 'echo never > /sys/kernel/mm/transparent_hugepage/enabled' as root, and add it to your /etc/rc.local in order to retain the setting after a reboot. Redis must be restarted after THP is disabled.
+    web_1    |  * Debugger PIN: 330-787-903
+    redis_1  | 1:M 17 Aug 22:11:10.483 * Ready to accept connections
+    ```
+
+    Compose pulls a Redis image, builds an image for your code, and start the
+    services you defined. In this case, the code is statically copied into the image at build time.
+
+2.  Enter `http://0.0.0.0:5000/` in a browser to see the application running.
```

Comment:
Is there any reason to list the `http://0.0.0.0:5000` option first? I think it's more intuitive if we talked about `http://localhost:5000` first.